grails-app/conf/runtime.groovy

```diff
@@ -137,7 +137,7 @@ openboxes {
             defaultLabel = "Analytics"
             menuItems = [
                 // TODO: Add option to include label 'beta'
-                [label: "inventory.browse.label", defaultLabel: "Browse Inventory", href: "/inventoryBrowser/index"],
+                [label: "inventory.view.label", defaultLabel: "Inventory Browser", href: "/inventoryBrowser/index"],
```

Comment:
`inventory.view.label` is `View Inventory`m your default is `Inventory Browser`, which one you are expecting here? If the later one, then add a separate message property, if the former one, then change the default label